CRM/Speakcivi/Page/Optout.php

```diff
@@ -0,0 +1,91 @@
+<?php
```

Comment:
refactoring, it's a bit too much of duplication with the confirm, most of the logic can be put in a parent class